cpp/harvest/nacjd.cc

```diff
@@ -178,14 +180,20 @@ void ExtractIDsFromWebsite(const std::set<std::string> &parsed_marc_ids, unsigne
     std::ofstream json_new_titles(NACJD_NEW_TITLES_JSON);
     json_new_titles << "{ \"nacjd\" : [ " << '\n';
     bool first(true);
+    unsigned int limit_test(0);
```

Comment:
limit_test seems to be debug-related and should be removed

Reply:
if you wanna keep it, please rename it and change the type from `unsigned int` to `unsigned`